tensorflow/lite/kernels/internal/reference/integer_ops/softmax.h

```diff
@@ -20,10 +20,21 @@ limitations under the License.
 namespace tflite {
 namespace reference_integer_ops {
 
-// Quantized softmax with int8 input and output.
+template<typename T> struct OutputParams {};
+template<> struct OutputParams<int8> {
+  static const int output_num_bits = 8;
+  static const int32 output_min = std::numeric_limits<int8>::min();
+};
+template<> struct OutputParams<int16> {
+  static const int output_num_bits = 16 - 1;
```

Comment:
What is the output scale of the int16 softmax?

Reply:
1 / pow(2, 15)